azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -873,6 +919,21 @@ func flattenAzureRmCosmosDBAccountCapabilities(capabilities *[]documentdb.Capabi
 	return &s
 }
 
+func flattenAzureRmCosmosDBAccountVirtualNetworkRules(rules *[]documentdb.VirtualNetworkRule) *schema.Set {
+	results := schema.Set{
+		F: resourceAzureRMCosmosDBAccountVirtualNetworkRuleHash,
+	}
+
+	for _, r := range *rules {
```

Comment:
We should probably be checking if rules is nil before dereferencing it

Reply:
Done.